utils/mstpctl-utils-interfaces.5

```diff
@@ -53,8 +53,8 @@ iface br0 inet static
     mstpctl_portrestrrole swp1=no swp2=no
     mstpctl_bpduguard swp1=no swp2=no
     mstpctl_portrestrtcn swp1=no swp2=no
-    mstpctl_treeprio 32768
-    mstpctl_treeportprio swp3=128
+    mstpctl_treeprio 8
```

Comment:
let's make this commit part of an initial set of commits;
the patch to `rpm.spec` may move to commit https://github.com/mstpd/mstpd/pull/25/commits/9e9639fa662e011a6d3970f04275760b9be9eeb0

the rest would be handled by the rename commits